tensorflow/core/kernels/non_max_suppression_op_test.cc

```diff
@@ -355,6 +355,178 @@ TEST_F(NonMaxSuppressionV2OpTest, TestEmptyInput) {
   test::ExpectTensorEqual<int>(expected, *GetOutput(0));
 }
 
+#if GOOGLE_CUDA
```

Comment:
Existing BUILD target doesn't run gpu tests, please move these test cases to a new `non_max_suppression_op_gpu_test.cc` file and add a tf_cuda_cc_test target for it.